src/transformers/models/roberta/modeling_roberta.py

```diff
@@ -1341,3 +1342,154 @@ def create_position_ids_from_input_ids(input_ids, padding_idx):
     mask = input_ids.ne(padding_idx).int()
     incremental_indices = torch.cumsum(mask, dim=1).type_as(mask) * mask
     return incremental_indices.long() + padding_idx
+
+@add_start_docstrings(
+    """
+    RoBERTa Model transformer with a span classification head on top (a linear layer on top of the
+    pooled output) e.g. for SuperGLUE WiC task.
+    """,
+    ROBERTA_START_DOCSTRING,
+)
```

Comment:
Here we should add this line: # Copied from transformers.models.bert.modeling_bert.BertSelfAttention with Bert->Roberta

Reply:
This way the BERT code is automatically and correct copied to RoBERTa :-)